### DISCUSSION TITLE
I use a plugin which uses proposal-optional-chaining, and project can't be built on node v12

### DISCUSSION THREAD

**Discussion by dimaninc2** [vercel/next.js#30784] [Help]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

12.19.0

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next build

### Describe the Bug

I use a plugin for photo gallery, it uses modern syntax with proposal optional chaining. My nextjs project can't be built:

```
> Build error occurred
/Projects/Webstorm/s-frontend/node_modules/@splidejs/react-splide/dist/js/react-splide.cjs.js:2479
    this.splide?.go(control);
                ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:931:16)
    at Module._compile (internal/modules/cjs/loader.js:979:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)
    at Module.load (internal/modules/cjs/loader.js:879:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:903:19)
    at require (internal/modules/cjs/helpers.js:74:18)
```

I've installed `@babel/plugin-proposal-optional-chaining` and created `.babelrc` with the following contents:
```
{
  "presets": [
    "next/babel"
  ],
  "plugins": [
    "@babel/plugin-proposal-optional-chaining"
  ]
}
```

and this didn't help

I've read that nextjs v11 uses webpack 5, but i can't build the project

### Expected Behavior

Proposal optional chaining syntax to be understood.

### To Reproduce

npm i @splidejs/react-splide @splidejs/splide --save
npm i @babel/plugin-proposal-optional-chaining --save-dev
next build

**Comment by timneutkens** [comment]
Next.js already supports optional chaining but node_modules are not compiled using Babel.

**Reply by dimaninc2** [reply]
and how can i compile node_modules using babel?

**Reply by zanzlender** [reply]
I know I'm a little late but I found the answer in Babel's docs: [@babel/plugin-proposal-optional-chaining](https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining)

In short, you install it with
```
npm install --save-dev @babel/plugin-proposal-optional-chaining
```

Then create a **.babelrc** config file in the root and add this: 
```JSON
{
  "plugins": ["@babel/plugin-proposal-optional-chaining"]
}
```
